app/views/shared/_footer.html.erb

```diff
@@ -1,5 +1,5 @@
-<footer class="footer">
-  <div class="container">
+<footer class="footer p-2">
+  <div class="container footer-container">
```

Comment:
How about changing this class to **"footer-container"** instead of **"container footer-container"**? I don't see any container class used in its scc file.